code/modules/mob/living/carbon/superior_animal/robots/subtype/greyson/types/mech.dm

```diff
@@ -0,0 +1,105 @@
+/mob/living/carbon/superior_animal/robot/greyson/synthetic/ripley
+	name = "Greyson Positronic Ripley unit"
+	desc = "An AI controlled autonomous power loading unit equipped with a diamond tipped drill."
+	icon_state = "greyson_ripley"
+	attacktext = "drilled"
+	health = 500
+	maxHealth = 500
+	melee_damage_lower = 40
+	melee_damage_upper = 45
+	attack_sound = 'sound/mecha/mechdrill.ogg'
+	flash_resistances = 100 // Robot
+
+/mob/living/carbon/superior_animal/robot/greyson/synthetic/ripley/slip(var/slipped_on,stun_duration=8)
+	return FALSE
+// Big stompers can't be slipped!
+
+/mob/living/carbon/superior_animal/robot/greyson/synthetic/ripley/attack_hand(mob/living/carbon/M as mob)
+	..()
+	var/mob/living/carbon/human/H = M
+
+	switch(M.a_intent)
+		if (I_HELP)
+			help_shake_act(M)
+
+		if (I_GRAB)
+			if(!weakened && stat == CONSCIOUS)
+				M.adjustBruteLoss(25)
+				M.adjustBruteLoss(25)
+				M.adjustBruteLoss(25)
+				M.adjustBruteLoss(25)
+				M.adjustBruteLoss(25)
+				M.adjustBruteLoss(25)
+				M.adjustBruteLoss(25)
+				M.adjustBruteLoss(25)
+				M.adjustOxyLoss(25)
+				M.Weaken(5)
+				visible_message(SPAN_WARNING("\red [src] immediately crushes [M] with its titan bulk when they stupidly try to grab it!"))
```

Comment:
```suggestion
				visible_message(SPAN_WARNING("\red [src] immediately crushes [M] with its titan bulk when they stupidly try to grab it!"))
```
What the hell is the \red here for? This is probably something that's already been here but I'm confused.

Reply:
Actually this probably just makes the text red and I'm an idiot, aren't I?

Reply:
litearlly no idea

Reply:
It is for the red text, yes